src/ucp/core/ucp_context.c

```diff
@@ -1721,6 +1727,8 @@ static ucs_status_t ucp_fill_config(ucp_context_h context,
         goto err_free_alloc_methods;
     }
 
+    context->config.ext.rndv_frag_mem_type = config->rndv_frag_mem_type;
```

Comment:
can we save the value in `context->config.ext.` directly?
i. e. would need to define the var, like:
`ucs_offsetof(ucp_config_t, ctx.rndv_frag_mem_type)`

Reply:
unhandled

Reply:
@brminich Sorry I missed this but address the specific comment in the last commit.